library/cassandra/dashboards/cassandra.json

```diff
@@ -0,0 +1,326 @@
+                            }
+                        ]
+                    }
+                },
+                {
+                    "visualization": {
+                        "id": "viz.line"
+                    },
+                    "layout": {
+                        "column": 1,
+                        "row": 19,
+                        "height": 3,
+                        "width": 4
+                    },
+                    "title": "Memtable Sizes",
+                    "rawConfiguration": {
+                        "nrqlQueries": [
+                            {
+                                "accountId": 0,
+                                "query": "SELECT average(`db.allMemtablesOnHeapSizeBytes`) * uniqueCount(entityName) AS 'On Heap', average(`db.allMemtablesOffHeapSizeBytes`) * uniqueCount(entityName) AS 'Off Heap' FROM CassandraSample TIMESERIES UNTIL 1 minute AGO"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "visualization": {
+                        "id": "viz.line"
+                    },
+                    "layout": {
+                        "column": 5,
+                        "row": 19,
+                        "height": 3,
+                        "width": 4
+                    },
+                    "title": "Avg Pending Compactions",
+                    "rawConfiguration": {
+                        "nrqlQueries": [
+                            {
+                                "accountId": 0,
+                                "query": "SELECT average(`db.pendingCompactions`) * uniqueCount(entityName) * uniqueCount(`db.columnFamily`) FROM CassandraColumnFamilySample FACET `db.keyspace` TIMESERIES UNTIL 1 minute AGO"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "visualization": {
+                        "id": "viz.area"
+                    },
+                    "layout": {
+                        "column": 9,
+                        "row": 19,
+                        "height": 3,
+                        "width": 4
+                    },
+                    "title": "Hints In Progress, by node",
+                    "rawConfiguration": {
+                        "nrqlQueries": [
+                            {
+                                "accountId": 0,
+                                "query": "SELECT average(`db.totalHintsInProgress`)  * uniqueCount(entityName) AS 'In Progress' FROM CassandraSample FACET entityName TIMESERIES UNTIL 1 minute AGO"
+                            }
+                        ]
+                    }
+                }
+            ]
+        }
+    ],
+    "name": "Cassandra",
```

Comment:
Move `name` and `description` to top of JSON file.

Reply:
done :)